### ISSUE TITLE
New Tab Does Not Open When Clicking on an External/Internal Link

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#156]
Screengrab/videos (insert file names here)
2020 08 26, New Tab Does Not Open When Clicking on an Internal Link.mp4

Link
https://d8-stg.boston.gov/departments/environment/understanding-plastic-bag-ordinance

Node # (if known)
[New Tab Does Not Open When Clicking on an Internal Link.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5132896/New.Tab.Does.Not.Open.When.Clicking.on.an.Internal.Link.mp4.zip)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOs 10.15.5 > Chrome 84

Trusted Tester Test
N/a

Brief Description
When clicking on an internal link on the boston.gov website, a new tab does not open. To get back to the previous page, you would need to get onto the back button, and this would be very hard to do, if not impossible, with a keyboard only. 

Steps to reproduce

1)	00:00, Go to the site, using the link above. 
2)	00:13, Click on the link: Read the Official Ordinance online.
3)	00:22, A PDF version of the Ordinance appears from the same tab that was opened on a different page on Boston.gov. While this is not specifically addressed in Sec 508 conformance, the user has lost Focus and will need to find the link again and launch it to go back to the previous page they were on. The only other option is to use the back button, but it is very difficult, if not impossible, to get to back button and launch and click the icon with only keyboard access.
a.	Note, if Iterators were testing the site for Functional testing only, we would flag this as a bug.  Given that Accessibility is being tested under Sec 508 conformance, this specific issue is not specifically However, its confusing and Focus is lost. Also, a person using the mouse could accomplish the task quickly while using only the keyboard will take substantially more time.
b.	Note: There are many places on the website where links are opened in this same manner while other links are opened in a new tab. For uniformity this should be corrected with someone making a list of all the links that should be fixed. If we need to do this at some point, please let us know.
Desired Result
•	Able to click on an internal link within the Boston.gov website and keep the original link open while viewing the new information in a new tab.

Actual Result
•	Unable to click on an internal link, within the Boston.gov website, and keep the original link open.

**Comment by subaha-cob** [comment]
@jill-iterators We have decided that because of the way our users navigate our website it will not make sense to make all our internal links open in a new tab. There are too many really and the users will loose track of the pages pretty quickly. They also have the option to use the back button with no issues. But external links on the other hand we will enforce the new window/tab rules on. Also the internal links opening in a new window were all done deliberately for those links. Please let us know your questions or concerns.

@duffy-james any thoughts on this?

**Comment by ollie-iterators** [comment]
This is done, as referenced above
